umh-core/pkg/config/dataflowcomponentconfig/config.go

```diff
@@ -53,6 +57,9 @@ func (bc *BenthosConfig) ToBenthosServiceConfig() benthosserviceconfig.BenthosSe
 
 // GetBenthosServiceConfig converts the component config to a full BenthosServiceConfig
 func (c *DataFlowComponentConfig) GetBenthosServiceConfig() benthosserviceconfig.BenthosServiceConfig {
+	if c == nil {
```

Comment:
I dont think this works.
If c is nil, you cannot call c.GetBenthosServiceConfig